Modules/Core/SpatialObjects/include/itkPointBasedSpatialObject.h

```diff
@@ -66,6 +66,10 @@ class ITK_TEMPLATE_EXPORT PointBasedSpatialObject:
    * each point (for computing world coordinates) */
   virtual void AddPoint( const SpatialObjectPointType & newPoints );
 
+  /** Assign points to this object, and assigned this object to
```

Comment:
Documentation wrong. Copy-pasted but not modified?

Reply:
Thanks!  Fixed!